lib/oneview-sdk/resource/api300/synergy/server_profile.rb

```diff
@@ -72,6 +72,18 @@ def self.get_sas_logical_jbod_attachment(client, name)
           results = get_sas_logical_jbod_attachments(client)
           results.find { |attachment| attachment['name'] == name }
         end
+
+        # Sets the OS deployment settings applicable when deployment is invoked through server profile
+        # @param [OneviewSDK::API300::Synergy::OSDeploymentPlan] os_deployment_plan the OSDeploymentPlan resource with valid URI
+        # @param [Array<Hash<String, String>>] custom_attributes The custom attributes to be configured on the OS deployment plan
+        # @option custom_attributes [String] :name name of attribute
```

Comment:
Try renaming `custom_attributes` to  `custom_attributes.entries` this way it won't confuse who's reading the documentation.